app/utils/FastlyUtils.scala

```diff
@@ -3,10 +3,17 @@ package utils
 import com.gu.i18n.CountryGroup
 import play.api.mvc.Request
 
-object RequestCountry {
-  implicit class RequestWithFastlyCountry(r: Request[_]) {
-    def getFastlyCountry = r.headers.get("X-GU-GeoIP-Country-Code").flatMap(CountryGroup.byFastlyCountryCode)
+object FastlyUtils {
+
+  implicit class FastlyRequest(r: Request[_]) {
+
+    def getFastlyCountryCode: Option[String] = r.headers.get("X-GU-GeoIP-Country-Code")
```

Comment:
Interesting to note that the `X-` prefix we use for this one is deprecated: https://tools.ietf.org/html/rfc6648 - wouldn't say it's worth changing though.